### ISSUE TITLE
Change SCM commit keybinding

### ISSUE THREAD

**Issue by lengk** [Microsoft/vscode#42862]
In the source control window you can press `cmd + enter` to commit the code to the repo. Is there a way of changing this keybinding?

Does this issue occur when all extensions are disabled?: Yes

**Comment by joaomoreno** [comment]
It's currently hardcoded. Would you like to customize it?

**Comment by lengk** [comment]
Yes I would like to use the keybinding somewhere else.

**Comment by joaomoreno** [comment]
What do you mean somewhere else? Like, from the editor? You can already do that by configuring a keybinding to the `git.commit` command.

**Comment by lengk** [comment]
No I mean the shortcut in the SCM Sidebar.

![SCM Sidebar](https://user-images.githubusercontent.com/5440566/35928804-fb66eff6-0c25-11e8-8310-fcc1c1bac4df.png)

**Comment by joaomoreno** [comment]
> Yes I would like to use the keybinding somewhere else.

So, you meant "Yes I would like to use another keybinding", right? 👍

**Comment by lengk** [comment]
Yeah I would like to change it :)

**Comment by lengk** [comment]
I'm using the keybind to start debugging. So when i'm using the SCM window, it starts debugging

**Comment by joaomoreno** [comment]
Actually I just gave this a try. If you customise a keybinding for the `git.commit` command, that will work even inside the commit text box. Did you try that yet?

**Comment by lengk** [comment]
Yes thank you @joaomoreno  :)

However when trying this before I didn't thin it would work as the text in the commit message box didn't change

**Comment by lengk** [comment]
Also the shortcut cmd+enter still works when in the scm menu

**Comment by joaomoreno** [comment]
> However when trying this before I didn't thin it would work as the text in the commit message box didn't change

The text goes away. Doesn't it go away on your machine?

> Also the shortcut cmd+enter still works when in the scm menu

Is that really still an issue for you?

**Comment by lengk** [comment]
> The text goes away. Doesn't it go away on your machine?

No the text doesn't go away on my machine?

![screen shot 2018-02-08 at 14 21 19](https://user-images.githubusercontent.com/5440566/35977564-5fa27746-0cdb-11e8-8468-57691446d926.png)

> Is that really still an issue for you?

Yes, as I said I'm running the program with cmd+enter. I could change it from that but vscode should really allow for that customisation

**Comment by joaomoreno** [comment]
I assigned <kbd>Ctrl 0</kbd> in my machine. When I use it here's the behaviour:

![kapture 2018-02-08 at 16 37 09](https://user-images.githubusercontent.com/22350/35981765-5fdfa702-0cee-11e8-86d1-7afe9c7f3983.gif)

Are you sure the input isn't cleared once the commit happens? Can you show me in a screencast?

> Yes, as I said I'm running the program with cmd+enter. I could change it from that but vscode should really allow for that customisation

But Cmd+Enter will only commit if you are focused in the commit box. Do you often start debugging using Cmd+Enter, when focused in the commit box?

**Comment by lengk** [comment]
Yeah it works on my computer too as you show but CMD+Enter still works as well. I want to stop command+enter working.

Yes I often press `cmd + enter` in the commit window and in most parts of the application
